podcasts/End of Year/Stories/EpilogueStory.swift

```diff
@@ -77,13 +77,15 @@ struct EpilogueStory: ShareableStory {
                     .onAppear(perform: prepareHaptics)
                 )
 
-                VStack {
-                    Spacer()
-                    HStack {
-                        Spacer()
-                        Image("logo")
-                            .padding(.bottom, geometry.size.height * 0.06)
+                if !renderForSharing {
+                    VStack {
                         Spacer()
+                        HStack {
```

Comment:
I also added this since is a small change.

To test:

1. Go to the last story
2. Trigger a screenshot
3. Tap "Share"
4. Save the image
5. ✅ Verify there is just a single logo at the bottom